08-regression2.Rmd

```diff
@@ -2,7 +2,7 @@
 
 ## Overview 
 Up until this point, we have solved all of our predictive problems---both classification
-and regression---using K-nearest neighbours-based approaches. But while K-NN methods are 
+and regression---using K-nearest neighbours (K-NN)-based approaches. But while K-NN methods are 
 simple and flexible, in the context of regression, there is another method (generally more popular in
 most scientific disciplines) known as *linear regression*. This chapter provides an introduction
```

Comment:
since this is just the overview, not going to make people read through an argument for linear regression here. saying interpretable isn't great here because that forces us to explain why. i'd rather leave the argument/explanation for the intro (not the overview).

but edited to get rid of the "popular" statement